fields/types/list/ListType.js

```diff
@@ -101,7 +103,7 @@ list.prototype.addToSchema = function () {
 			);
 		}
 		fields[path] = createField(path, fieldsSpec[path]);
-		fieldsArray.push(path);
+		paths.push(path);
```

Comment:
Oh - I renamed it because `fieldArray` was actually an array of names and not the actual fields like in the `List` class.